isis/src/cassini/apps/cisscal/tsts/badCisscalImage/Makefile

```diff
@@ -0,0 +1,9 @@
+APPNAME = cisscal
+
+include $(ISISROOT)/make/isismake.tsts
+
+
+commands:
+	-$(APPNAME) from=$(INPUT)/N1605851822_1.cub \
```

Comment:
What does the `-` in front of the `$` do here?

Reply:
It ignores the exit code from cisscal since it technically errors and throws an exception

Reply:
Thanks!